tests/30rooms/14forget.pl

```diff
@@ -0,0 +1,278 @@
+test "Forgotten room messages cannot be paginated",
+   requires => [ local_user_and_room_fixtures(), local_user_fixture() ],
+
+   do => sub {
+      my ( $creator, $room_id, $user ) = @_;
+
+      matrix_join_room( $user, $room_id )
+      ->then( sub {
+         matrix_send_room_text_message( $creator, $room_id, body => "sup" )
+      })->then( sub {
+         do_request_json_for( $user,
+            method => "GET",
+            uri    => "/api/v1/rooms/$room_id/messages",
+            params => {
+               limit => 1,
+               dir   => 'b'
+            },
+      )})->then( sub {
+         my ( $body ) = @_;
+
+         require_json_keys( $body, qw( chunk ) );
+         log_if_fail "Chunk", $body->{chunk};
+         $body->{chunk}[0]{content}{body} eq "sup" or die "Wrong message";
+
+         Future->done(1);
+      })->then( sub {
```

Comment:
You claim 'Done' but the lines are still present here. Did you forget to commit/push it, or is github confused?